core/server/master/src/test/java/alluxio/master/file/DefaultFileSystemMasterTest.java

```diff
@@ -1570,9 +1571,9 @@ private void startServices() throws Exception {
     mRegistry = new MasterRegistry();
     JournalFactory factory = new MutableJournal.Factory(new URI(mJournalFolder));
     mBlockMaster = new BlockMaster(mRegistry, factory);
-    mExecutorService =
-        Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("FileSystemMasterTest-%d", true));
-    mFileSystemMaster = new FileSystemMaster(mRegistry, factory,
+    mExecutorService = Executors
+        .newFixedThreadPool(2, ThreadFactoryUtils.build("DefaultFileSystemMasterTest-%d", true));
+    mFileSystemMaster = new DefaultFileSystemMaster(mRegistry, factory,
```

Comment:
I see ... what about creating two separate test files ... one that tests the `DefaultFileSystemMaster` specific logic (which at the moment is only the `stop()` test and one that tests `FileSystemMaster` logic (all other tests)